acceptance_tests/features/environment.py

```diff
@@ -12,6 +14,8 @@
 
 logger = wrap_logger(logging.getLogger(__name__))
 
+register_type(Boolean=lambda text: strtobool(text))
```

Comment:
👍 I'd like to make more use of this feature to help keep out step code simpler

I think the typical naming convention would be lower case for the type name though, like `boolean`